### PR TITLE
Align HPO search space to new default hyper parameter

### PR DESCRIPTION
**Pull request by eunwoosh**
This PR includes aligning HPO search space to new default hyper parameters.

### CODE REVIEW

**Comment by eunwoosh** [comment]
run ote-test

**Review by yunchu — APPROVED** [review]

**Review by jaegukhyun — APPROVED** [review]

File: external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ hp_space:
...
-      - 0.1
+      - 0.0014
+      - 0.035
       - 0.0001
   learning_parameters.batch_size:
     param_type: qloguniform
     range:
-      - 64
-      - 256
+      - 42
```
Comment (JihwanEom) [inline external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml:13]: Is it typo of 32?

File: external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml [hunk 2]
```diff
@@ -4,12 +4,12 @@ hp_space:
   learning_parameters.learning_rate:
     param_type: qloguniform
     range:
-      - 0.0003
-      - 0.1
+      - 0.0014
```
Comment (JihwanEom) [inline external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml:7]: Could you revert learning rate as default value of template.yaml? We discarded bigger learning rate training schedule by performance degradation for C-IL experiments.

File: external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/hpo_config.yaml [hunk 3]
```diff
@@ -4,12 +4,12 @@ hp_space:
...
-      - 0.08
+      - 0.0008
+      - 0.1
       - 0.0001
   learning_parameters.batch_size:
     param_type: qloguniform
     range:
-      - 40
-      - 96
+      - 64
```
Comment (JihwanEom) [inline external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/hpo_config.yaml:13]: Could you explain why only mbv3 starts from 64 bs?

File: external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml (hunk 1 above)
Comment (eunwoosh) [reply external/model-preparation-algorithm/configs/classification/efficientnet_b0_cls_incr/hpo_config.yaml:13]: No it's correct.

File: external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/hpo_config.yaml (hunk 3 above)
Comment (eunwoosh) [reply external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/hpo_config.yaml:13]: I conducted experiments for 3 cases and selected best case. So each models can have different search space.

**Review by JihwanEom — APPROVED** [review]
LGTM
